gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/api/Plugin.java

```diff
@@ -52,4 +52,9 @@ public interface Plugin {
      * Plugin dependencies.
      */
     URL[] dependencies();
+
+    /**
+     * Plugin deployment status.
+     */
+    boolean deployed();
```

Comment:
shouldn't we add a default implementation to return true? Or are we sure that every implementation of Plugin interface will be updated?

My concern is "what if I run an APIM platform using this version of gravitee-plugin, but with implementations classes that don't?"

Do you see what I mean? Or maybe my question does not make sense here 😄?

Reply:
Interesting. I tend to agree with Rémi and go without a default implementation to force a proper implementation. Most of the implementations are located in gravitee-plugin or in the apim or am repository directly.

Reply:
Ok, I thought it was also in every plugins.

Reply:
For info, I tested gravitee-node with this PR and no issue.